widgets/BigQueryDelete-action.json

```diff
@@ -0,0 +1,70 @@
+{
+  "metadata": {
+    "spec-version": "1.5"
+  },
+  "display-name" : "Google BigQuery Delete",
+  "configuration-groups": [
+    {
+      "label" : "Project",
```

Comment:
Please be consistent with the configuration groups used in all the other plugins, with 'Basic', 'Credentials', and 'Advanced' groups.